### ISSUE TITLE
Make LOV entrance

### ISSUE THREAD

**Issue by chile12** [HajoRijgersberg/OM#10]
I would urge you to invest some time in the dissemmination of your very helpful ontology, to increase visibility and usability:

* Suggest a new entrance for Linked Open Vocabularies (https://lov.linkeddata.es/dataset/lov/suggest)
* Follow the metadata guidlines, especially using vann: (https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf)
* http://www.ontology-of-units-of-measure.org is missing a link to the whole ontology resource (at least I could not find it in a 1 minute search). Ideally this would be reachable under the namespace (in different serializations with content nagotiation):
*   http://www.ontology-of-units-of-measure.org/page/om-2.owl
*   http://www.ontology-of-units-of-measure.org/page/om-2.ttl
*   http://www.ontology-of-units-of-measure.org/page/om-2.nt

**Comment by HajoRijgersberg** [comment]
Finally found time to work on your raised issues.

> Suggest a new entrance for Linked Open Vocabularies
I have just tried to suggest 'http://www.ontology-of-units-of-measure.org/page/om-2' there, but an error is being returned. Could you perhaps help me out; what I'm doing wrong?

> Follow the metadata guidlines, especially using vann
I plan to make work of this issue next week. I'll keep you up to date.

> http://www.ontology-of-units-of-measure.org is missing a link to the whole ontology resource
On that page, 'OM 2: Units of Measure' is listed, among other ontologies, in the left pane. Do you perhaps mean that 'http://www.ontology-of-units-of-measure.org/page/om-2.owl' and the .ttl and .nt should be mentioned explicitly there?

**Comment by HajoRijgersberg** [comment]
Thanks for the hints.
Indeed, if after processing the changes it still doesn't work to suggest the ontology, I'll contact the people of LOV.

**Comment by jmkeil** [comment]
Very good idea. But be careful regarding naming clashes with the [ISO 19156 Observation Model (om)](https://lov.linkeddata.es/dataset/lov/vocabs/om). And please let me hint at [WIDOCO](https://github.com/dgarijo/Widoco), which might help with these issues, again (#1).

**Comment by HajoRijgersberg** [comment]
Good point. But do you mean that the namespace abbreviation 'om' of OM should be changed? I'm afraid to do that, afraid of any (unforeseen) consequences of that.

**Comment by HajoRijgersberg** [comment]
Dear people,
I thought I could manage the metadata quickly, but it appears to take more time. So I'll look for a new moment next week.

**Comment by HajoRijgersberg** [comment]
Dear people,

It is my honour and pleasure to announce that Bhoomin Pandya has submitted OM to LOV, in a successful way! Please see the above-mentioned issue (https://github.com/HajoRijgersberg/OM/issues/36#).
She has also written a nice message about it in Medium: https://link.medium.com/fVXqSdNNo8.
Thank you very much, Bhoomin! :)

Best, Hajo

**Comment by conceptminers1** [comment]
Many Thanks Hajo.
Wishing Best to You and Team.
